pkgs/development/libraries/gcr/default.nix

```diff
@@ -83,17 +85,22 @@ stdenv.mkDerivation rec {
     # We are still using ssh-agent from gnome-keyring.
     # https://github.com/NixOS/nixpkgs/issues/140824
     "-Dssh_agent=false"
+  ] ++ lib.optionals (!stdenv.isLinux) [
+    "-Dsystemd=disabled"
   ];
 
   doCheck = false; # fails 21 out of 603 tests, needs dbus daemon
 
   PKG_CONFIG_SYSTEMD_SYSTEMDUSERUNITDIR = "${placeholder "out"}/lib/systemd/user";
 
+  # FIXME: drop meson_post_install.py on 3.90+
```

Comment:
GCR 4 will be a different package since they do not have compatible ABI. Also no need to specify fixme’s that will break loudly on update.

Reply:
Fixme removed. Thanks!